.github/workflows/all_green_check.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: all_green
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - labeled
+      - unlabeled
```

Comment:
this triggers are impacting `changelog` workflow only, we should create a different file for that and all_green does not need to changelog being success as this will fail automatically
anyway we can keep it like that as it is already like this in the main branch